openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -56,29 +56,19 @@ def _do_third_party_auth(request):
             u"with backend_name {backend_name}".format(
                 username=username, backend_name=backend_name)
         )
-        message = _(
+        message = Text(_(
             u"You've successfully logged into your {provider_name} account, "
-            u"but this account isn't linked with an {platform_name} account yet."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += _(
+            u"but this account isn't linked with an {platform_name} account yet. {br_tags}"
             u"Use your {platform_name} username and password to log into {platform_name} below, "
-            u"and then link your {platform_name} account with {provider_name} from your dashboard."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += Text(_(
+            u"and then link your {platform_name} account with {provider_name} from your dashboard.{br_tags}"
             u"If you don't have an {platform_name} account yet, "
             u"click {register_label_strong} at the top of the page."
         )).format(
+            br_tags=HTML('<br/><br/>'),
             platform_name=platform_name,
+            provider_name=requested_provider.name,
             register_label_strong=HTML('<strong>{register_text}</strong>').format(
-                register_text=_('Register')
+                register_text=Text(_('Register'))
```

Comment:
`Text()` is not needed here (although it won't hurt).  Any variables sent to `HTML().format()` or `Text().format()` will be escaped, unless they are wrapped again with `HTML()`.